dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -2454,9 +2454,16 @@ public ProgramOutput stopServer(boolean postStopServerArchive, boolean forceStop
 
             int serverStopRC = output.getReturnCode();
             if (serverStopRC != 0) {
-                throw new RuntimeException("Server stop failed with RC " + serverStopRC +
-                                           ".\nStdout:\n" + output.getStdout() +
-                                           "\nStderr:\n" + output.getStderr());
+                if (serverStopRC >= 20) {
```

Comment:
only do additional processing if the RC was >= 20, anything less and there's probably a legitimate problem, so lets save the processing cycles.